### DISCUSSION TITLE
Smart Meter only and no sensor appear

### DISCUSSION THREAD

**Discussion by PatriceHenriet** [thomluther/ha-anker-solix#132] [Q&A] [answered]
The Anker Solix Integration for Home Assistant is correctly installed, see a Smart Meter but create none sensor

is there a need to do something ?

Regards

**Comment by thomluther** [comment, marked as answer]
Hello
You need to add the smart meter to a system in the App, otherwise you will not see much via the Integration since stand alone devices do NOT report any usage data to the Cloud Api server. They use only the MQTT servers for real time consumption data while the App is watching their data.

**Reply by PatriceHenriet** [reply]
Ok thanks
